app/api/helpers/scheduled_jobs.py

```diff
@@ -160,6 +160,18 @@ def expire_pending_tickets():
         db.session.commit()
 
 
+def event_invoices_mark_due():
+    from app import current_app as app
+    with app.app_context():
+        db.session.query(EventInvoice).filter(EventInvoice.status == 'upcoming',
```

Comment:
Shift filter to next line and unindent each line

Reply:
@iamareebjamal Made the requested change.